data/json/monsters/feral_humans.json

```
@@ -95,7 +97,71 @@
     "melee_dice": 2,
     "melee_dice_sides": 7,
     "melee_cut": 9,
-    "death_drops": "feral_humans_death_drops_axe"
+    "death_drops": "feral_humans_death_drops_axe",
+    "fungalize_into": "mon_feral_human_axe_fungal_infected"
+  },
+  {
+    "id": "mon_feral_human_pipe_fungal_infected",
+    "type": "MONSTER",
+    "description": "Delicate fungal stalks sprout in rows from sores in the arms, and the facial expression is pure joy.  Coos softly.",
+    "copy-from": "mon_feral_human_pipe",
+    "default_faction": "fungus",
```

```suggestion
    "default_faction": "fungus",
```
These should still be identical to the original ferals at this point. This phase is the same as a player that has been infected with spores and doesn't know it. From a gameplay perspective, a player shouldn't be able to tell that anything has happened to the feral at all, just like they can't tell when they've been infected with fungal spores.

---

Yep, that was an error